pkg/project/registry/project/proxy/proxy.go

```diff
@@ -128,6 +128,29 @@ func (s *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	return convertNamespace(namespace), nil
 }
 
+func (s *REST) Update(ctx kapi.Context, obj runtime.Object) (runtime.Object, bool, error) {
```

Comment:
> we need a test to make sure a project admin can't update their project/namespace until we get per-field restrictions in place (to prevent them changing node selectors or scc uid allocations)

Well, there's a pickle.  Existing policy today will let a project admin update a project.  Even if its updated, unless the policy is updated before the update is applied and the API server is started (currently not possible), then we can't safely fix it.  How frustrating.